### PR TITLE
Adding Bittrex.com and Aeria Games

### PR DESCRIPTION
**Pull request by GuardianLiarus**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -298,6 +298,16 @@
         ]
     },
 
+    {
+        "name": "Aeria Games",
+        "url": "https://www.aeriagames.com",
+        "difficulty": "easy",
+        "notes": "Login, go to your profile settings on the top right, then go to the 'Account' tab, and then click 'Delete my Account' ",
```
Comment (tupaschoal) [inline _data/sites.json:305]:
```suggestion
        "notes": "Login, go to your profile settings on the top right, then go to the 'Account' tab and click 'Delete my Account'",
```

File: _data/sites.json [hunk 2]
```diff
@@ -1196,6 +1206,16 @@
         ]
     },
 
+    {
+        "name": "Bittrex",
+        "url": "https://bittrex.zendesk.com/hc/en-us/requests/new?ticket_form_id=114093958171",
+        "difficulty": "Hard",
```
Comment (tupaschoal) [inline _data/sites.json:1212]:
```suggestion
        "difficulty": "hard",
```

File: _data/sites.json [hunk 3]
```diff
@@ -1196,6 +1206,16 @@
...
+        "notes": "The only way to delete an account is to send a customer support request, select 'Close account' and fill the form, you will then recieve an email ironically asking you to create a Bittrex support account password so you can follow the status of your request.",
+        "domains": [
+            "global.bittrex.com",
```
Comment (tupaschoal) [inline _data/sites.json:1215]:
```suggestion
            "global.bittrex.com"
```

File: _data/sites.json [hunk 4]
```diff
@@ -1196,6 +1206,16 @@
         ]
     },
 
+    {
+        "name": "Bittrex",
+        "url": "https://bittrex.zendesk.com/hc/en-us/requests/new?ticket_form_id=114093958171",
+        "difficulty": "Hard",
+        "notes": "The only way to delete an account is to send a customer support request, select 'Close account' and fill the form, you will then recieve an email ironically asking you to create a Bittrex support account password so you can follow the status of your request.",
```
Comment (tupaschoal) [inline _data/sites.json:1213]:
```suggestion
        "notes": "The only way to delete an account is to send a customer support request, select 'Close account' and fill the form, you will then receive an email ironically asking you to create a Bittrex support account password so you can follow the status of your request.",
```

**Review by tupaschoal — COMMENTED** [review]
A few light edits to fix the formatting and improve the wording.

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Comment by GuardianLiarus** [comment]
> A few light edits to fix the formatting and improve the wording.

Damnit, every time i think i did a proper contribution someone has to fix my mistakes 😅, thanks!

**Comment by tupaschoal** [comment]
Don't worry at all! This happens to everyone and it's normal :)

**Pull request merged**